### PR TITLE
[SKIP CI] Add known issues to vFile document.

### LINKED ISSUE #1678: Document that storage on local driver is not supported in kahuna
The vanilla docker volumes are not supported as kahuna backends

### LINKED ISSUE #1680: Document Kahuna's interoperability with multi tenancy

### LINKED ISSUE #1898: vFile: Add a known issue about how to clean up the volume after failed mounting
After volume mount failed, the "Volume Status" will in "Error" status.  User cannot recover from this state, and need to remove this volume completely. We need to document this as a known issue.

### PR DESCRIPTION
**Pull request by lipingxue**
Fixed #1678 
Fixed #1680 
Fixed #1898

### CODE REVIEW

**Comment by shuklanirdesh82** [comment]
@lipingxue Adding `[SKIP CI]` to this PR, please make sure to add as part of initial commit message while raising a PR/merging to master.

Thanks!

File: docs/external/vfile-plugin.md [hunk 1]
```diff
@@ -160,4 +186,30 @@ This error message indicates the swarm cluster is not in a healthy status.
...
     2. An `N` manager cluster will tolerate the loss of at most `(N-1)/2` managers.
-    3. Docker recommends a maximum of seven manager nodes for a swarm.
\ No newline at end of file
+    3. Docker recommends a maximum of seven manager nodes for a swarm.
+
+
+### Does vFile plugin support multi-tenancy feature?
+No. vFile plugin does not support multi-tenency, which is currently an experimental feature for vDVS.
+Assume we have created a vmgroup ```vmgroup1``` and add VM "node1" to ```vmgroup1```. VM "node2" has not been added to any vmgroup and is considered belong to ```_DEFAULT``` vmgroup.
+Then create a volume "vol1" from "node02" and a volume "vol2" from "node01" using vFile plugin.
```
Comment (shuklanirdesh82) [inline docs/external/vfile-plugin.md]: ```node02``` => node2 and same for node1 too

File: docs/external/vfile-plugin.md [hunk 2]
```diff
@@ -160,4 +186,30 @@ This error message indicates the swarm cluster is not in a healthy status.
...
+```
+vmware@ubuntu16:~$ docker volume ls
+DRIVER              VOLUME NAME
+vsphere:latest      _vF_vol2@vsanDatastore
+vfile:latest        vol1
+vfile:latest        vol2
+
+```
+
+On node 02, we are also able to see both "vol1" and "vol2".
```
Comment (shuklanirdesh82) [inline docs/external/vfile-plugin.md]: VM name reference should be consistent.

File: docs/external/vfile-plugin.md [hunk 3]
```diff
@@ -150,6 +153,29 @@ docker plugin install --grant-all-permissions --alias vfile vmware/vfile:latest
 
 This will increase timeout to 90 sec, from default of 30 sec.
 
+### Volume mount failed, retry mount with the same volume still failed.
+Check the "Volume Status" field of ``` docker volume inspect ```, if the "Volume Status" field shows "Error", which means volume is in error status, user need manually remove the volume to recover.
```
Comment (shuklanirdesh82) [inline docs/external/vfile-plugin.md]: ```user need manually remove the volume to recover.``` <= should be reworded, it's confusing.

File: docs/external/vfile-plugin.md (hunk 1 above)
Comment (lipingxue) [reply docs/external/vfile-plugin.md]: Done.

File: docs/external/vfile-plugin.md (hunk 2 above)
Comment (lipingxue) [reply docs/external/vfile-plugin.md]: Done.

File: docs/external/vfile-plugin.md (hunk 3 above)
Comment (lipingxue) [reply docs/external/vfile-plugin.md]: Could you elaborate why it is confusing?

**Pull request merged**